Add information about on demand repository cleaning (#1267)
theforeman/foreman-documentation: guides/common/modules/proc_reclaiming-space-from-on-demand-repositories.adoc

[id="reclaiming-space-from-on-demand-repositories_{context}"]
= Reclaiming Space From On Demand Repositories

> If you set  the _download policy_ to on demand, {Project} downloads packages only when the clients request them.

Reviewer: ```suggestion
If you set the _download policy_ to on demand, {Project} downloads packages only when the clients request them.
```

[id="reclaiming-space-from-on-demand-repositories_{context}"]
= Reclaiming Space From On Demand Repositories

> When the download policy is set to on demand, the {Project} downloads packages only when the clients request them.

Reviewer: ```suggestion
If you set the download policy to _on demand_, {Project} downloads packages only when clients request them.
```

> When the download policy is set to on demand, the actual packages are downloaded only when the clients request them.

Reviewer 2: Is actual needed here ?
```suggestion
When the download policy is set to on demand, the packages are downloaded only when the clients request them.
```

> These packages can be cleaned up to reclaim space.

Reviewer 2: Rephrase this to avoid the passive voice.

.For a single repository

> 

Reviewer: ```suggestion
```

You don't need an empty live below headings starting with a dot. Please fix all instances.
